docs/self-hosted/troubleshooting.md

```diff
@@ -110,8 +110,29 @@ update the host IP and the NO_PROXY environment.
 
 #### Reset Replicated Management Console authentication
 
-If you cannot log into the Replicated Management Console anymore, then you can reset LDAP and password authentication on the host system by running the following command:
+If you cannot log into the Replicated Management Console anymore, then you can
+reset LDAP and password authentication on the host system by running the
+following command:
 
 ```
 replicatedctl console-auth reset
 ```
+
+#### Issues with Multi-factor Authentication
+
+If you have set up MFA and you can't log in with your generated codes, there may
+be a time-drift issue with either the Self-hosted Private Packagist server or
+the device you are using to generate the codes.
```

Comment:
This is also an issue when trying to set up MFA in the first place if the server time is off